components/form/RadioButton .js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import PropTypes from 'prop-types';
+import { ErrorMessage } from '.';
+import { Label } from './Label';
+
+export function Radio({ name, label, value }) {
+  const { register } = useFormContext();
+  return (
+    <>
+      <div className="flex items-center my-1 space-x-1 text-[20px] mr-2 ">
+        <input
+          className="cursor-pointer"
+          id={`radio-${value}`}
+          name={name}
+          {...register(name)}
+          type="radio"
+          value={value}
+        />
+        <Label name={name} htmlFor={`radio-${value}`} text={label} />
+      </div>
+    </>
+  );
+}
+
+Radio.propTypes = {
+  label: PropTypes.node.isRequired,
+  name: PropTypes.string,
+  value: PropTypes.any
+};
+Radio.defaultProps = {
+  name: null,
+  value: null
+};
+export function RadioButton({ column, name, options }) {
```

Comment:
This should be RadioGroup since it is made up of multiple Radio Buttons:

```suggestion
export function RadioGroup({ column, name, options }) {
```